libs/ui/src/lib/build-status/build-status.component.ts

```diff
@@ -206,58 +200,57 @@ export class BuildStatusComponent {
     })
   );
 
-  readonly chunks$ = this.status$.pipe(
+  readonly bundles$ = this.status$.pipe(
     map(status => {
-      if (status && status.stats && status.stats.chunks) {
-        return status.stats.chunks;
+      if (status && status.stats && status.stats.bundles) {
+        return status.stats.bundles;
       } else {
         return [];
       }
     }),
-    tap(chunks => {
-      const defaultChunk = chunks.find(x => /main/.test(x.name)) || chunks[0];
-      if (defaultChunk) {
-        this.currentChunkId$.next(defaultChunk.id);
+    tap(bundles => {
```

Comment:
Hmm, it'd have to be a subject or a class property since the user can select the the bundle using the dropdown in UI.

One way to simplify (maybe) is to make this an explicit subscription instead of using `tap`.